cfn-resources/project-invitation/test/inputs_1_invalid.template.json

```diff
@@ -1,11 +1,8 @@
 {
-    "ApiKeys": {
-        "PublicKey": "",
-        "PrivateKey": ""
-    },
-    "ProjectId": "",
-    "Username": "",
-    "Roles": [
-        "GROUP_DATA_ACCESS_ADMINjs"
-    ]
+  "Profile": "default",
+  "ProjectId": "",
+  "Username": "",
+  "Roles": [
+    "GROUP_DATA_ACCESS_ADMINjs"
```

Comment:
(Also sorry for the late spot, just caught it when double checking)

Reply:
The test is for invalid template, so this would be on purpose

Reply:
Doh! Yes, great I understand now.

Nit in future - if we're testing for invalid, it's best to make it clear with an obviously wrong value, rather than one that looks like a typo.